src/js/components/settings/integrations.js

```diff
@@ -139,21 +143,51 @@ export const Integrations = ({ integrations = [], changeIntegration, createInteg
   };
 
   return (
-    <div>
-      <h2 className="margin-top-small">Integrations</h2>
-      {!isCreating && !!availableIntegrations.length && (
-        <Select displayEmpty onChange={onConfigureIntegration} value={currentValue} style={{ minWidth: 300 }}>
-          <MenuItem value="">Add new integration</MenuItem>
-          {availableIntegrations.map(item => (
-            <MenuItem key={item.provider} value={item.provider}>
-              {item.title}
-            </MenuItem>
+    <div style={{ minHeight: '95%', display: 'flex', flexDirection: 'column' }}>
+      <div>
+        <h2 className="margin-top-small">Integrations</h2>
+        {!isCreating && !!availableIntegrations.length && (
+          <Select displayEmpty onChange={onConfigureIntegration} value={currentValue} style={{ minWidth: 300 }}>
+            <MenuItem value="">Add new integration</MenuItem>
+            {availableIntegrations.map(item => (
+              <MenuItem key={item.provider} value={item.provider}>
+                {item.title}
+              </MenuItem>
+            ))}
+          </Select>
+        )}
+        {configuredIntegrations.map(integration => (
+          <IntegrationConfiguration key={integration.id} integration={integration} onCancel={onCancelClick} onDelete={deleteIntegration} onSave={onSaveClick} />
+        ))}
+      </div>
+      <div style={{ marginTop: 'auto' }}>
+        <Divider />
+        <div className="margin-top-small margin-bottom-small">
+          {[azureIoT].map((tile, index) => (
+            <img key={`tile-${index}`} src={tile} />
           ))}
```

Comment:
```suggestion
          <img src={azureIoT} />
```
unless you expect there to be more images listed in the future that is 👌